keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/ScalarDoubleTensor.java

```diff
@@ -358,6 +359,17 @@ public double min() {
         return value;
     }
 
+    @Override
+    public IntegerTensor argMax() {
+        return IntegerTensor.scalar(0);
+    }
+
+    @Override
+    public IntegerTensor argMax(int axis) {
+        TensorShapeValidation.checkDimensionExistsInShape(axis, this.getShape());
+        return IntegerTensor.create(0).reshape(1);
```

Comment:
This looks weird. ND4J doesn't support rank 1's (vectors). Why doesn't this just return an `int` anyway?